shoes-swt/lib/shoes/swt/common/painter.rb

```diff
@@ -98,7 +98,7 @@ def reset_rotate(transform, graphics_context, angle, left, top)
           transform.translate left, top
           transform.rotate angle
           transform.translate(-left, -top)
-          graphics_context.setTransform transform
+          graphics_context.set_transform transform
```

Comment:
Hm I see that the other cases also seem to use `set_` so I'd make this an all or nothing thing, doesn't have to hold up this PR though